test/integration/test-submit-custom-header.js

```diff
@@ -83,16 +74,12 @@ server.listen(common.port, function() {
       'x-test-header': 'test-header-value'
     }
   }, function(err, res) {
-
-    if (err) {
-      throw err;
-    }
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Expected '{' and instead saw 'throw'.](https://www.codacy.com/app/form-data/form-data/file/1700090166/issues/source?bid=2805153&fileBranchId=3045625#l77)